src/main/java/com/uber/jenkins/phabricator/conduit/DifferentialClient.java

```diff
@@ -114,8 +114,22 @@ public JSONObject postComment(String message) throws IOException, InterruptedExc
     }
 
     protected JSONObject callConduit(String methodName, Map<String, String> params) throws IOException, InterruptedException, ArcanistUsageException {
-        ArcanistClient arc = new ArcanistClient(this.arcPath, "call-conduit", params, this.conduitToken, methodName);
+        ArcanistClient arc = getArcanistClient(methodName, params);
         return arc.parseConduit(this.launcher.launch(), this.launcher.getStderr());
```

Comment:
I'm ok with either. We should agree on a format. If we do launcher.launch(), launcher must be a local variable in the method scope. If it's a variable at the class level, maybe we should adopt the member variable naming scheme (eg. mLauncher at the class level). Let's not solve that in this PR though.